search/views/mobile_search_clients_engines_sources_daily.view.lkml

```diff
@@ -1,10 +1,21 @@
 include: "//looker-hub/search/views/mobile_search_clients_engines_sources_daily.view.lkml"
 
 view: +mobile_search_clients_engines_sources_daily {
-  dimension: month_formatted {
-    type: date
-    sql: DATE(2020, EXTRACT(MONTH FROM CAST(${TABLE}.submission_date AS DATE)), 1) ;;
-    html: {{ rendered_value | date: "%b" }} ;;
```

Comment:
Probably a better idea to leave this until it's confirmed the `month_name` is appropriate